src/components/MyKiva/BorrowerCarousel.vue

```diff
@@ -298,6 +319,93 @@ const loadMore = () => {
 	router.push('/portfolio/loans');
 };
 
+const goToLink = () => {
+	$kvTrackEvent('borrower-profile', 'go-to-old-bp', undefined, `${this.selectedLoan?.id}`);
+	window.open(`lend/${this.selectedLoan?.id}`);
+};
+
+const handleCloseSideSheet = () => {
+	showSideSheet.value = false;
+	selectedLoan.value = undefined;
+};
+
+const addToBasket = lendAmount => {
```

Comment:
This code is duplicated from LBC, right? Would it make sense to move the `KvSideSheet` and `BorrowerSideSheetContent` into a shared component, and then move this util method there as well?